modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/groupby/BaseGroupByTestCase.java

```diff
@@ -189,23 +187,22 @@ public void testGroupByDocsSortsSortFieldDesc() throws Exception {
 
 	@Test
 	public void testGroupByDocsStart() throws Exception {
-		Map<String, Integer> map1 = HashMapBuilder.put(
+		Map<String, String> map1 = new HashMap<>();
+
+		Map<String, Integer> map2 = HashMapBuilder.put(
```

Comment:
@julschong Why swapping map1 and map2?

Reply:
@dantewang I'm trying to keep map2 declaration and usage closer together

Reply:
maybe I'm overthinking this